src/test/java/StringManipulationTest.java

```diff
@@ -20,65 +22,104 @@ public void tearDown() {
     }
 
     @Test
+    // Checks behavior for a string
```

Comment:
//I think this test works, I'm not quite sure why it's not showing up green but I would try something like this: 
@Test
public void testCount1() {
    manipulatedstring.setString("This is my string");
    int length = manipulatedstring.count();
    assertEquals(17, length);
}